content/how-to/dedicated-hosting/how-to-request-a-csr/index.md

```diff
@@ -18,24 +18,30 @@ Request (CSR).
 
 1. Log in to [MyRackspace Portal](https://login.rackspace.com/login) with your username and password.
 
-2. On the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="logincredentials" src="/support/how-to/how-to-request-a-csr/csr1.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+2. On the top navigation bar, choose **Support** > **Support Center**.
 
-4. Select the **Subject** field.
+   <img width="941" alt="selectsupportcenter" src="/support/how-to/how-to-request-a-csr/csr2.png">
 
-5. From the drop-down menu, select **Generate Certificate Signing Request (CSR)**.
+3. Under **Common Request** menu, click **See All Templates**. The **See All Templates** displays list of all templates.
 
-   {{<image src="CSR1.png" alt="" title="">}}
+   <img width="958" alt="templatelist" src="/support/how-to/how-to-request-a-csr/csr3.png">
```

Comment:
Highlight and merge images in Paint and Power Point

Reply:
Check with Steven.